src/components/card/CartCard.jsx

```diff
@@ -1,14 +1,40 @@
 import React from 'react';
+import { lakersCity } from '../../assets';
 import classes from './CartCard.module.css';
+import { useCart } from '../../context/cart/cart-context';
+import { useProducts } from '../../context/product-listing/products-context';
+
+const CartCard = (product) => {
+  const {
+    id,
+    image = lakersCity,
+    title = 'Lakers City Edition',
+    itemDescription = 'Nike Dri-FIT NBA Swingman Jersey',
+    quantity = 1,
+    price = 1000,
+    wishlisted = false,
+  } = product;
+
+  console.log(product);
+
+  const { cartDispatch } = useCart();
+  const { productsDispatch } = useProducts();
+
+  const itemQuantityChangeHandler = (e) => {
+    // Network call here to change quantity then the following actions
```

Comment:
If you want to add comments for better understanding then write them as JS docs. Maybe [this](https://www.section.io/engineering-education/jsdoc-documentation/) could help

Reply:
Thank you for this